spec/features/sign_in_spec.rb

```diff
@@ -12,17 +12,17 @@
       user.confirm
       page.fill_in 'user_email', with: user.email
       page.fill_in 'user_password', with: '123456'
-      click_button  "Sign in"
+      click_button  'Sign in'
```

Comment:
You can create a method `fill_form_and_click` with the email and password as params, so you can reuse the code in all the tests. Something like:
```
def fill_form_and_click(email, password)
  page.fill_in 'user_email', with: email
  page.fill_in 'user_password', with: password
  click_button  'Sign in'
end
```